src/TrackGenerator.cpp

```diff
@@ -439,12 +441,12 @@ void TrackGenerator::setNumAzim(int num_azim) {
  * @brief Set the suggested track spacing (cm).
  * @param spacing the suggested track spacing
  */
-void TrackGenerator::setTrackSpacing(double spacing) {
+void TrackGenerator::setDesiredAzimSpacing(double spacing) {
   if (spacing < 0)
     log_printf(ERROR, "Unable to set a negative track spacing %f for the "
```

Comment:
If we are using "azimuthal spacing" in place of "track spacing" now, then this error message should be changed to reflect that.